pkg/model/keycloak_deployment.go

```diff
@@ -19,12 +21,56 @@ const (
 	ProbeTimeBetweenRunsSeconds = 30
 )
 
+const (
+	KeycloakPodDefaultMemoryRequests = "600Mi"
```

Comment:
Let's remove the default. As you pointed out in the JIRA (or Keycloak-dev, I don't quite remember) the default might be different depending on the Providers you're using.

Reply:
ok